### ISSUE TITLE
Broken with djangorestframework==3.8.2

### ISSUE THREAD

**Issue by zulrang** [barseghyanartur/django-elasticsearch-dsl-drf#50]
```
lib/python3.6/site-packages/django_elasticsearch_dsl_drf/pagination.py", line 13, in <module>
    from rest_framework.pagination import _get_count
ImportError: cannot import name '_get_count'

```

**Comment by zulrang** [comment]
Fixed in pull request https://github.com/barseghyanartur/django-elasticsearch-dsl-drf/pull/51

**Comment by barseghyanartur** [comment]
Released in 0.7.2.
